javascript-light-bulb/main.js

```diff
@@ -0,0 +1,14 @@
+var $clickBackground = document.querySelector('.background-color');
+var $clickLightbulb = document.querySelector('.lightbulb');
+
+function clickOn(event) {
+  if ($clickLightbulb.className === 'lightbulb-off') {
```

Comment:
By doing this, your logic is dependent on the current state of the DOM, which is not ideal. Instead, I would create a variable to keep track of if the light is currently on or not, and use that variable's value to determine what action to take when the light is clicked.